.gitignore

```diff
@@ -1,6 +1,9 @@
 # All built stuff
 /out/
 
+# Cache stuff
+__pycache__
+
```

Comment:
https://github.com/github/gitignore/blob/main/Python.gitignore

I usually opt for `curl https://raw.githubusercontent.com/github/gitignore/main/Python.gitignore >> .gitignore` -- you can get the raw link from the github page. There's usually some editing to do afterwards as well, they have comments explaining when you may want to ignore particular things.